compiler-rt/lib/builtins/CMakeLists.txt

```diff
@@ -237,6 +237,14 @@ if(NOT FUCHSIA AND NOT COMPILER_RT_BAREMETAL_BUILD)
   )
 endif()
 
+option(COMPILER_RT_LIBATOMIC_USE_PTHREAD
```

Comment:
Perhaps this can be the default for Linux if not Android.

Reply:
We can try to switch that after this PR lands.